### PR TITLE
Planet carousel

### PR DESCRIPTION
**Pull request by LaurencePortron**
Carousel ready for merge

### CODE REVIEW

**Review by PN-TH — CHANGES_REQUESTED** [review]
Erreur dans la console:
Warning: A string ref, "carouselViewport", has been found within a strict mode tree. String refs are a source of potential bugs and should be avoided. We recommend using useRef() or createRef() instead.

**Comment by LaurencePortron** [comment]
Error has been fixed, no console errors left

File: src/components/home/Carousel.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  };
+
+  checkNumberOfSlidesToScroll() {
+    let numberOfSlidesToScroll;
+    if (window.innerWidth <= 900) {
+      numberOfSlidesToScroll = 1;
+    } else {
+      numberOfSlidesToScroll = 1;
+    }
+    // eslint-disable-next-line react/destructuring-assignment
```
Comment (ComicScrip) [inline src/components/home/Carousel.js:52]: A corriger

File: src/components/home/Carousel.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
+/* eslint-disable react/no-string-refs */
```
Comment (ComicScrip) [inline src/components/home/Carousel.js:1]: à corriger

**Review by ComicScrip — APPROVED** [review]

**Pull request merged**